jparest_settings.xml

```diff
@@ -3,15 +3,9 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <servers>
         <server>
-            <id>callisto-jparest</id>
-            <configuration>
-                <httpHeaders>
-                    <property>
-                        <name>Authorization</name>
-                        <value>Bearer ${ARTIFACTORY_TOKEN}</value>
-                    </property>
-                </httpHeaders>
-            </configuration>
+            <id>github</id>
+            <username>DummyUser</username>

```

Comment:
Is it worth adding a comment here:
`<!-- Any username would work as long as it's not empty -->`

Reply:
Comment addded.